packages/x-tailwindcss/src/x-tailwind-plugin/components/sliding-panel/index.ts

```diff
@@ -1,16 +1,18 @@
 import { TailwindHelpers } from '../../../types';
 import { slidingPanelDefault } from './default';
+import { slidingPanelFade } from './fade';
 
 /**
  * Returns the component `sliding panel` CSS.
  *
- * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @param helpers
```

Comment:
Es lint is complaining without this 👀 
```suggestion
 * @param helpers - The {@link TailwindHelpers} to generate CSS.
```